README.md

```diff
@@ -23,33 +23,45 @@ If you'd like to contribute to `modlunky2` here are some steps to setup your env
 ### Creating VirtualEnv
 In the root directory you can make a virtualenv. It will be excluded from commits by default
 ```console
-> python -m venv venv
+python -m venv venv
```

Comment:
Why did you remove the console marker? You didn't explain anywhere why you made this change and it wasn't applied consistently as you let them in the setup section. I tend to prefer a console character to distinguish where input goes in an example.